src/middlewares/task.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response, NextFunction } from 'express'
+import HttpError from '../utils/HttpError'
+import mongoose from 'mongoose'
+
+export default {
+  errorHandlerMiddleware (error: HttpError, req: Request, res: Response, next: NextFunction): void {
+    const status = error.status || 500
+    const message = error.message || 'Something went wrong '
+    res
+      .status(status)
+      .send({
+        status,
```

Comment:
You don't need to send status here too

Reply:
fixed e1098e4418555c67b50397f454c93ff0a4312a66